src/plone/restapi/serializer/summary.py

```diff
@@ -83,7 +83,10 @@ def __init__(self, context, request):
         self.blocklisted_attributes = metadata["blocklisted_attributes"]
 
     def __call__(self):
-        obj = IContentListingObject(self.context)
+        try:
+            obj = IContentListingObject(self.context)
+        except TypeError:
+            return {}
```

Comment:
Doesn't this just make it less obvious what the problem is? As a consumer of the API, I would find it surprising to see an empty object in a list of items, and I would wonder what happened. Better to throw the error that happens now, I think.